portal-impl/src/portal.properties

```diff
@@ -6982,6 +6987,11 @@
     #
     feature.flag.LPD-31212.type=release
 
+    #
+    # Env: LIFERAY_FEATURE_PERIOD_FLAG_PERIOD__UPPERCASEL__UPPERCASEP__UPPERCASED__MINUS__NUMBER3__NUMBER4__NUMBER6__NUMBER3__NUMBER6__PERIOD_TYPE
+    #
+    feature.flag.LPD-34636.type=beta
+
```

Comment:
This should be a development flag, not a beta flag. A beta flag is customer-facing, appears in UI. It would need a description lang key, so SF will fail on this solution. A developer flag is just in code, doesn't require lang keys. You can just remove this block.

Reply:
Thank you! I've just remove it